cocotb_build_libs.py

```diff
@@ -19,6 +20,96 @@
 cocotb_share_dir = os.path.abspath(os.path.join(os.path.dirname(__file__), "cocotb", "share"))
 
 
+def create_sxs_assembly_manifest(name, filename, libraries):
+    """
+    Create side-by-side (sxs) assembly manifest
+    It contains dependencies to other assemblies (in our case the assemblies are equal to the other libraries)
+    For more details see:
+     - https://docs.microsoft.com/en-us/windows/win32/sbscs/assembly-manifests
+     - https://docs.microsoft.com/en-us/windows/win32/sbscs/using-side-by-side-assemblies
+
+    `name` is the name of the assembly for which the manifest is generated, e.g. `libcocotbutils`
+    `filename` is the filename of the library, e.g. `libcocotbutils.dll`
+    `libraries` is a list of names of dependent manifests, e.g. `["libgpilog"]`
```

Comment:
In case we ever build sphinx docs from this:

```suggestion
    Create side-by-side (sxs) assembly manifest
    
    It contains dependencies to other assemblies (in our case the assemblies are equal to the other libraries).
    For more details see:
     - https://docs.microsoft.com/en-us/windows/win32/sbscs/assembly-manifests
     - https://docs.microsoft.com/en-us/windows/win32/sbscs/using-side-by-side-assemblies

    Args:
        name: The name of the assembly for which the manifest is generated, e.g. ``libcocotbutils``.
        filename: The filename of the library, e.g. ``libcocotbutils.dll``.
        libraries: A list of names of dependent manifests, e.g. ``["libgpilog"]``.
```